Test/civl/inductive-sequentialization/snapshot-scatter-gather-full.bpl

```diff
@@ -0,0 +1,234 @@
+// RUN: %parallel-boogie "%s" > "%t"
```

Comment:
Should we drop ```-full``` from the name?

Reply:
yes, it is not full yet. I plan to make it full later, but even then we don't need full in the name.